src/main/Kotlin/HelloWorld.kt

```diff
@@ -0,0 +1,5 @@
+fun main(args: Array<String>) {
```

Comment:
Die Ordnerstruktur heißt üblicherweise `src/main/kotlin` nicht `.../Kotlin` (auf nicht Windows-Systemen macht Groß-/Kleinschreibung einen Unterschied). Da du in der POM `<sourceDirectory>src/main/kotlin</sourceDirectory>` schreibst und ich Linux verwende (Travis ebenso) funktioniert der Build bei mir dementsprechend nicht korrekt.